test/ziggurat/middleware/default_test.clj

```diff
@@ -82,29 +81,27 @@
 
 (deftest protobuf->hash-test-alpha-and-deprecated
   (testing "Deprecated protobuf deserializer"
```

Comment:
why do we need the test for deprecated serializer now?

Reply:
removed..